test/package.spec.ts

```diff
@@ -2,20 +2,50 @@
  * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
  */
 
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import Package from '../src/commands/package';
+import { getCleanProjectDir, getTestOutputDir } from './test-utils';
+
+const testOutputDir = getTestOutputDir(__filename);
+const nestAppDir = path.resolve('test', 'nest');
+
 describe('Package', () => {
-  it('should copy files correctly without parameters', () => {
-    expect(true).toBeTruthy();
+  afterAll(() => {
+    // fs.removeSync(testOutputDir);
   });
 
-  it('should copy files correctly with custom globs', () => {
-    expect(true).toBeTruthy();
+  it('should copy files correctly without parameters', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'no-params');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['package.json', 'package-lock.json']);
   });
 
-  it('should overwrite output folder', () => {
-    expect(true).toBe(true);
+  it('should copy files correctly with custom globs', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'globs');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--include=*.json', '--exclude=package*,tsconfig*', '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['nest-cli.json', 'tslint.json']);
   });
 
-  it('should install productive dependencies only', () => {
-    expect(true).toBe(true);
+  it('should overwrite output folder', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'folder-overwrite');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--include=.gitignore', '--skipInstall']);
+    await Package.run([projectDir, '--include=README.md', '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toEqual(['README.md']);
```

Comment:
As above, I would've expect the test to assert that `.gitignore` does not exist in the output directory.

Reply:
This test uses `.toEqual()` and therefore check if the directory ONLY includes the `README.md`

Reply:
Ah, that escaped me, thanks!